pwned_passwords/pwned-passwords.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+set -eu
+
+usage() {
+  cat >&2 << EOM
+Usage: ${0} [-nufph]
+  -n : -n <number> Number of passwords to store. Default: 3000000
+  -u : -u <URL> URL for pwned passwords. Default: 'https://downloads.pwnedpasswords.com/passwords/pwned-passwords-sha1-ordered-by-count-v8.7z'
+  -f : -f <file> File to store pwned passwords. Default: pwned-passwords.txt
```

Comment:
if we moved the default definitions up we could reuse them here right?
```suggestion
  -n : -n <number> Number of passwords to store. Default: ${number_of_passwords}
  -u : -u <URL> URL for pwned passwords. Default: '${pwned_url}'
  -f : -f <file> File to store pwned passwords. Default: ${pwned_file}
```